src/pages/rss.xml.ts

```diff
@@ -0,0 +1,37 @@
+import { getRssString } from '@astrojs/rss';
+
+import { SITE, METADATA, APP_BLOG } from 'astrowind:config';
+import { fetchPosts } from '~/utils/blog';
+import { getPermalink } from '~/utils/permalinks';
+
+export const GET = async () => {
+  if (!APP_BLOG.isEnabled) {
+    return new Response(null, {
+      status: 404,
+      statusText: 'Not found',
+    });
+  }
+
+  const posts = await fetchPosts();
+
+  const rss = await getRssString({
+    title: `${SITE.name}’s Blog`,
```

Comment:
Not important, but while I'm here
```suggestion
    title: `${SITE.name} Blog`,
```